linera-service/tests/linera_net_tests.rs

```diff
@@ -2845,11 +2845,12 @@ async fn test_end_to_end_faucet(config: impl LineraNetConfig) -> Result<()> {
 #[cfg_attr(feature = "kubernetes", test_case(SharedLocalKubernetesNetTestingConfig::new(Network::Grpc, BuildArg::Build) ; "kubernetes_grpc"))]
 #[cfg_attr(feature = "remote-net", test_case(RemoteNetTestingConfig::new(None) ; "remote_net_grpc"))]
 #[test_log::test(tokio::test)]
+#[ignore = "This test takes a long time to run"]
 async fn test_end_to_end_faucet_with_long_chains(config: impl LineraNetConfig) -> Result<()> {
     let _guard = INTEGRATION_TEST_GUARD.lock().await;
     tracing::info!("Starting test {}", test_name!());
 
-    let chain_count = test_iterations().unwrap_or(1_000);
+    let chain_count = test_iterations().unwrap_or(10_000);
```

Comment:
I would have done it in a slightly different way:
* Run with say 30 entries in the regular CI by using the LINERA_TEST_ITERATIONS as it exerts the functionality, though I agree not completely.
* Run with 10000 in the main test done once in a while.

That being said, I am not the author of the test, so I leave it to the author to decide.

Reply:
Or the other way round:
* Make the default here 5, so that if you run `cargo test` you always also verify that we didn't break the test for some reason other than performance.
* Use `LINERA_TEST_ITERATIONS` in `long_faucet_chain_test.yml`.